examples/golang/components/sqldb-postgres-query/wasmcloud.toml

```
@@ -0,0 +1,9 @@
+name = "sqldb-postgres-query"
+version = "0.1.0"
+language = "tinygo"
+type = "component"
+
+[component]
+wit_world = "component"
```

```suggestion
wit_world = "component"
```
I think you can exclude this -- though I'm open to it staying for clarity

---

Apparently I do need it 😢 but perhaps we just need to adjust `wash build` for this?

```
missing `wit_world` in wasmcloud.toml ([component] section) for creating preview1 or preview2 components

```